src/img/africa.svg

```diff
@@ -1,45 +0,0 @@
-<svg width="240" height="240" viewBox="0 0 240 240" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
@MaximumCrash It looks like these files were deleted but not moved. I can't locate them in the `/public/images` folder after pulling down your branch and I'm getting a webpack build error:

```
 ERROR #98123  WEBPACK

Generating development JavaScript bundle failed

Can't resolve '@images/asia.svg' in '/Users/alexsingh/projects/maker/rejonPR/commdev-grants-transparency-dashboard/src/modules/Stats'

File: src/modules/Stats/Country.js
```

This error repeats for all the .svg files it cannot locate.

(Incidentally, we should probably be pulling out the svg code and creating components from it directly - but that's separate to this PR ;) )

Reply:
Ah my bad, it seems that the public file was located in the .gitignore file, but none of the file changes got updated! I've pushed these in now.